src/presign/participant.rs

```diff
@@ -1118,9 +1118,9 @@ impl PresignKeyShareAndInfo {
             let r2_pub_j = round_three_input.r2_public.clone();
             let r2_priv_j = round_three_input.r2_private.clone();
 
-            let alpha = BigNumber::from_slice(self.aux_info_private.sk.decrypt(&r2_pub_j.D.0)?);
+            let alpha = BigNumber::from_slice(self.aux_info_private.sk.decrypt(&r2_pub_j.D)?);
             let alpha_hat =
-                BigNumber::from_slice(self.aux_info_private.sk.decrypt(&r2_pub_j.D_hat.0)?);
+                BigNumber::from_slice(self.aux_info_private.sk.decrypt(&r2_pub_j.D_hat)?);
```

Comment:
note: I was trying to figure out why we have to use this `from_slice` externally to decryption and I decided we shouldn't. Made #101 to fix.